lib/src/client/call.dart

```diff
@@ -352,3 +367,23 @@ class ClientCall<Q, R> implements Response {
     } catch (_) {}
   }
 }
+
+/// Given a string, ensure the length is a multiple of 4, adding padding if needed.
+String padBase64Multiple4(String input) {
```

Comment:
Maybe `input.padRight((input.length + 3) & ~3, '=');` instead. 

I suggest just folding this into the caller function. It's not worth having a helper for that.

Reply:
Fixed in https://github.com/bottlepay/grpc-dart/commit/cd197160a01b0b1ac641eccd433d5b4f55f20f9c